### ISSUE TITLE
mvn gitflow:release - Not installing release-version

### ISSUE THREAD

**Issue by DemonicTutor** [aleksandr-m/gitflow-maven-plugin#13]
when running **mvn gitflow:release** i would expect the RELEASE to be installed in my local maven repository.

e.g.:
%MVN_REPO%/my/group/myartifact/1.0.0-SNAPSHOT/myartifact-1.0.0-SNAPSHOT.war
%MVN_REPO%/my/group/myartifact/1.0.0/myartifact-1.0.0.war

But this does not happen.. only the previous and next SNAPSHOT's are there.

my config:

``` xml
<plugin>
        <groupId>com.amashchenko.maven.plugin</groupId>
        <artifactId>gitflow-maven-plugin</artifactId>
        <configuration>
          <mvnExecutable>mvn</mvnExecutable>
          <gitExecutable>git</gitExecutable>

          <installProject>true</installProject>
          <verbose>true</verbose>

          <skipTag>false</skipTag>
          <keepBranch>false</keepBranch>
          <skipFeatureVersion>false</skipFeatureVersion>
          <skipTestProject>false</skipTestProject>
          <allowSnapshots>false</allowSnapshots>

          <gitFlowConfig>
            <productionBranch>master</productionBranch>
            <developmentBranch>develop</developmentBranch>
            <featureBranchPrefix>feature/</featureBranchPrefix>
            <releaseBranchPrefix>release/</releaseBranchPrefix>
            <hotfixBranchPrefix>hotfix/</hotfixBranchPrefix>
            <supportBranchPrefix>support/</supportBranchPrefix>
            <versionTagPrefix></versionTagPrefix>
          </gitFlowConfig>
          <commitMessages>
            <featureStartMessage>updating versions for feature branch</featureStartMessage>
            <featureFinishMessage>updating versions for development branch</featureFinishMessage>

            <hotfixStartMessage>updating versions for hotfix</hotfixStartMessage>
            <hotfixFinishMessage>updating for next development version</hotfixFinishMessage>

            <releaseStartMessage>release build ${project.version}</releaseStartMessage>
            <releaseFinishMessage>updating for next development version</releaseFinishMessage>

            <tagHotfixMessage>tagging hotfix</tagHotfixMessage>
            <tagReleaseMessage>tagging release</tagReleaseMessage>
          </commitMessages>
        </configuration>
      </plugin>
```

**Comment by aleksandr-m** [comment]
What is your use case for that? Why do you need a release version to be installed locally?

Unrelated: You do know that you don't have to put all plugin configuration option into pom, don't you?

**Comment by DemonicTutor** [comment]
We are not using a maven repo (artifactory / nexus) and i do not want to deploy to a server from maven.

After running the release i am SCPing the deployable artifacts (WAR, SQL-Scripts, properties) to a other site and runnig a automated deployment from there.

Currently after running **mvn gitflow:release** i have to checkout master and run **mvn clean install**.


Additionally i don't see a reason why you would not want to have the artifact installed which you just tried to create - probably unnecessary if most people run hudson and nexus but it still makes more sense to install locally too.

**Comment by DemonicTutor** [comment]
> Unrelated: You do know that you don't have to put all plugin configuration option into pom, don't you?

Yeah... that's just some kind of personal spleen with plugins i just started using... don't want to lookup all the options all the time but have them right there.

**Comment by aleksandr-m** [comment]
Installing next development version locally makes sense since most probably you're going to work with that. Deploying released artifact makes more sense than installing it locally, but that is another story. :)

Allowing to set profiles should be enough to execute configured plugins for that profile. (E.g. to install released artifact.)

Thanks for sharing.

**Comment by aleksandr-m** [comment]
@DemonicTutor Initial commit for allowing to run additional Maven goals is [here](https://github.com/aleksandr-m/gitflow-maven-plugin/commit/1e26c36077e91398638ba7b625810d55dc48aad8).
Please test and review.

**Comment by aleksandr-m** [comment]
Implemented in 9e57bb0.

**Comment by hfreedman** [comment]
Hello there, running `mvn -B gitflow:release`  does not deploy the non-snapshot version to nexus, so after it finishes, I do: `git checkout master && mvn deploy`. If I understand it right, using the -D postReleaseGoals=deploy will have the same result, but I did not. Would it be possible to update the usage in README file? Thanks a lot.
